composer.json

```diff
@@ -29,9 +29,10 @@
     "require-dev": {
         "sminnee/phpunit": "^5.7",
         "squizlabs/php_codesniffer": "^3",
-        "silverstripe/behat-extension": "^3",
+        "silverstripe/behat-extension": "^4",
         "silverstripe/serve": "^2",
-        "se/selenium-server-standalone": "2.41.0"
+        "se/selenium-server-standalone": "2.41.0",
+        "symbiote/silverstripe-queuedjobs": "^4"
```

Comment:
Is queued job needed for the behat test?

Reply:
No specifically for the behat test, though it makes sense in my head to be there for dev since it's in the "suggest" list already.  This is super minor though since we'll generally just use a recipe-testing type of thing anyway.

Reply:
My instinct would be to leave it out unless we're actually testing something that requires it or we expect that having install could break a use case in some way.

Reply:
Removed, added to travis as REQUIRE_EXTRA instead.  That's fairly standard, and seems good since no one outside of CMS squad or CI is every going to run behat